communities/pulumi.yml

```diff
@@ -0,0 +1,187 @@
+    - https://github.com/pulumi/facet
+    - https://github.com/pulumi/pulumi-mailgun
+    - https://github.com/pulumi/pulumi-minio
+    - https://github.com/pulumi/training
+    - https://github.com/pulumi/team-edition-for-open-source
+    - https://github.com/pulumi/upgrade-provider
+    - https://github.com/pulumi/pulumi-azure-serverless
+    - https://github.com/pulumi/ci-mgmt
+    - https://github.com/pulumi/pulumi-aws-quickstart-aurora-postgres
+    - https://github.com/pulumi/pulumi-demo-native
+    - https://github.com/pulumi/pulumi-cloud-import
+    - https://github.com/pulumi/pulumi-equinix-metal
+    - https://github.com/pulumi/homebrew-tap
+    - https://github.com/pulumi/pulumi-rke
+    - https://github.com/pulumi/templates-policy
+    - https://github.com/pulumi/pulumi-chocolatey
+    - https://github.com/pulumi/pulumi-package
+    - https://github.com/pulumi/spinnaker-plugins-repository
+    - https://github.com/pulumi/pulumi-azure-static-website
+    - https://github.com/pulumi/codeship-example
+    - https://github.com/pulumi/pulumi-iopipe
+    - https://github.com/pulumi/demo-kubecrash-2022
+    - https://github.com/pulumi/action-test-provider-downstream
+    - https://github.com/pulumi/aws-workshop
+    - https://github.com/pulumi/pulumi-metabase
+    - https://github.com/pulumi/pulumi-azure-quickstart-acr-geo-replication
+    - https://github.com/pulumi/pulumi-tls-self-signed-cert
+    - https://github.com/pulumi/pulumi-azure-justrun
+    - https://github.com/pulumi/jfrog-webinar
+    - https://github.com/pulumi/action-docker-build
+    - https://github.com/pulumi/pulumi-google-project-factory
+    - https://github.com/pulumi/aws-tf-fork-patch
+    - https://github.com/pulumi/guestbook
+    - https://github.com/pulumi/terraform
+    - https://github.com/pulumi/basic-example
+    - https://github.com/pulumi/build-automation
+    - https://github.com/pulumi/qcon-workshop
+    - https://github.com/pulumi/pulumi-aws-iam
+    - https://github.com/pulumi/terraform-provider-github
+    - https://github.com/pulumi/pulumi-google-cloud-static-website
+    - https://github.com/pulumi/pulumi-pascal
+    - https://github.com/pulumi/pulumi-import-google-cloud-account-scraper
+    - https://github.com/pulumi/pulumi-terraform-template
+    - https://github.com/pulumi/aws-proton-workshop-templates
+    - https://github.com/pulumi/terraform-diff-reader
+    - https://github.com/pulumi/grpc-debug-interceptors
+    - https://github.com/pulumi/pulumi-synced-folder
+    - https://github.com/pulumi/test-repo
+    - https://github.com/pulumi/frontend-working-group
+    - https://github.com/pulumi/blog-lambda-snapshots
+    - https://github.com/pulumi/pulumi-upgrade-provider-action
+    - https://github.com/pulumi/pulumi-hubspot
+    - https://github.com/pulumi/orbit-actions
+    - https://github.com/pulumi/pulumi-nodejs-dynamic
+  governance-repositories:
+    repo_urls:
+    - https://github.com/pulumi/pulumi
```

Comment:
Hi, @simenandre  thanks for your PR:

Normally, repositories in the two categories should not overlap. The former is used to store software artifacts source code, which are software components that accompany version releases, while the latter is used to store code repositories related to community governance, such as COCs, contributor guides, charters, etc.

Does this code repository at https://github.com/pulumi/pulumi have two classifications of attributes simultaneously?

BR//Yehui.

Reply:
Yes, I haven't found any other repository where governance is done, so I guess it has two. We could also remove from governance?

Reply:
Yes, you can remove it from governance type which is optional, otherwise data would be duplicated in general.

Reply:
```suggestion
```

Like this, or should we remove `governance-repositories` as well?